src/stk/utilities/utilities.py

```diff
@@ -449,6 +450,32 @@ def flatten(iterable, excluded_types=None):
             yield x
 
 
+def is_metal_atom(atom):
+    """
+    Check if atom has atomic number of a metal atom.
+
+    Parameters
+    ----------
+    atom : :class:`.Atom`
+        An stk Atom.
```

Comment:
no need to write stk here, because the type specification makes it clear that it is an stk atom.

Also if you were referring to stk somewhere in the docs, you would need to add markup.

Atom should also be lowercase.